src/BuildIt.Forms/BuildIt.Forms.Controls/CameraPreviewControl.xaml.cs

```diff
@@ -137,20 +79,34 @@ public CameraFacing PreferredCamera
         }
 
         /// <summary>
-        /// Toggles continuous auto-focus for selected camera device
+        /// Gets or sets the focus mode for the camera preview
         /// </summary>
-        public bool EnableContinuousAutoFocus
+        public CameraFocusMode FocusMode
         {
-            get => (bool)GetValue(EnableContinuousAutoFocusProperty);
-            set => SetValue(EnableContinuousAutoFocusProperty, value);
+            get => (CameraFocusMode)GetValue(FocusModeProperty);
+            set => SetValue(FocusModeProperty, value);
         }
 
+        /// <summary>
+        /// Gets or sets // TODO Add summary
+        /// </summary>
         public Aspect Aspect
         {
             get => (Aspect)GetValue(AspectProperty);
             set => SetValue(AspectProperty, value);
         }
 
+        /// <summary>
+        /// Gets or sets the command that will be executed every time the camera preview control got into an error state
+        ///
+        /// NOTE: When binding it in XAML, make sure that it's defined at the very top of the bindable properties
```

Comment:
Does this mean that this should be the first attribute defined on a XAML element? Is there a reason why this is required?

Reply:
Yes, because the issue might be related to setting the FocusMode property (e.g. desired focus mode not available) and we need to have the command already in place to notify user about the error